datasets/wit/README.md

```diff
@@ -0,0 +1,256 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- af
+- ar
+- ast
+- azb
+- be
+- bg
+- bn
+- br
+- ca
+- cs
+- cy
+- da
+- de
+- el
+- en
+- eo
+- es
+- et
+- eu
+- fa
+- fi
+- fr
+- fy
+- ga
+- gl
+- hr
+- hu
+- hy
+- id
+- it
+- iw
+- ja
+- ka
+- ko
+- la
+- lt
+- lv
+- mk
+- ml
+- ms
+- nl
+- nn
+- 'no'
+- pl
+- pt
+- ro
+- ru
+- sk
+- sl
+- sr
+- sv
+- th
+- tr
+- uk
+- ur
+- vi
+- vo
+- zh
+licenses:
+- cc-by-sa-3.0
+multilinguality:
+- multilingual
+paperswithcode_id: wit
+pretty_name: Wikipedia-based Image Text
+size_categories:
+- 10M<n<100M
+source_datasets:
+- original
+- extended|wikipedia
+task_categories:
+- text-retrieval
+- image-to-text
+task_ids:
+- text-retrieval-other-text-image-retrieval
+- image-captioning
+---
+
+# Dataset Card for [Dataset Name]
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/google-research-datasets/wit
+- **Repository:** https://github.com/google-research-datasets/wit
+- **Paper:** https://arxiv.org/abs/2103.01913
+- **Leaderboard:** https://www.kaggle.com/c/wikipedia-image-caption
+- **Point of Contact:** krishnaps@google.com
+
+### Dataset Summary
+
+Wikipedia-based Image Text (WIT) Dataset is a large multimodal multilingual dataset. WIT is composed of a curated set of 37.6 million entity rich image-text examples with 11.5 million unique images across 108 Wikipedia languages. Its size enables WIT to be used as a pretraining dataset for multimodal machine learning models.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
```

Comment:
Please add descriptions of the supported tasks here:
```
- `image-to-text`: ...
- `text-retrieval`: ...
```

Reply:
What do you mean? 
```
- `image-to-text`: Task is to generate text conditioned on an image
```